features/product/viewing_products/viewing_a_products_lowest_price_from_30_days_before_the_discount.feature

```diff
@@ -0,0 +1,58 @@
+@viewing_products
+Feature: Seeing the product's lowest price from 30 days before the discount
+    In order to know whether the current product's price discount is attractive
+    As a Guest
+    I want be be able to see a product's lowest price from 30 days before the discount
+
+    Background:
+        Given the store operates on a single channel in "United States"
+
+    @todo
+    Scenario: Not seeing the lowest price information on a product with no discount
+        Given the store has a product "Wyborowa Vodka" priced at "$21.00"
```

Comment:
It could have been moved to the background since we always need a product with a given price, and the discounts/price changes are either greater/lower than the base price, but for brevity, it should be fine to leave it as is.

Reply:
I agree that it could be in the background, however, it's easier to read all price setups in one place